test/validation/api/packet/packet.c

```diff
@@ -41,6 +41,25 @@ static struct udata_struct {
 	"abcdefg",
 };
 
+static struct {
+	odp_pool_t          pool;
+	odp_proto_chksums_t all_chksums;
+	int                 num_pkt;
+} parse_test;
+
+static uint32_t parse_test_pkt_len[] = {
+	sizeof(test_packet_arp),
+	sizeof(test_packet_ipv4_icmp),
+	sizeof(test_packet_ipv4_tcp),
+	sizeof(test_packet_ipv4_udp),
+	sizeof(test_packet_vlan_ipv4_udp),
+	sizeof(test_packet_vlan_qinq_ipv4_udp),
+	sizeof(test_packet_ipv6_icmp),
+	sizeof(test_packet_ipv6_tcp),
+	sizeof(test_packet_ipv6_udp),
+	sizeof(test_packet_vlan_ipv6_udp)
```

Comment:
could you please add IPsec packets from IPsec testsuite?

Reply:
Done in v2.